src/App.tsx

```diff
@@ -1,29 +1,109 @@
-import { useSelector } from 'react-redux';
+/* eslint-disable max-len */
+import {
+  FC, useEffect, useMemo, useState,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { useDispatch, useSelector } from 'react-redux';
 
-import { Start } from './components/Start';
-import { Finish } from './components/Finish';
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter';
+import { TodoModal } from './components/TodoModal';
+import { ComplitedFilter } from './types/ComplitedFilter';
+import './App.scss';
+import { Loader } from './components/Loader';
+import {
+  LOADING_SELECTORS, TODO_SELECTORS, RootState, TODOS_SELECTORS, TODOS_ACTIONS_CREATOR,
+} from './store';
 
-import { selectors } from './store';
+export const App: FC = () => {
+  const [searchQuery, setSearchQuery] = useState('');
+  const [complitedFilter, setComplitedFilter] = useState<ComplitedFilter>(ComplitedFilter.All);
 
```

Comment:
filters should place in redux

Reply:
Ok. Was the task updated? I didn't see it before.